ShortURLBuilderProvider/Sources/ShortURLBuilderProvider/Services/KeyChainService.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+import Security
+
+enum KeychainService {
+    static func save(_ value: String, forKey key: String) -> Bool {
+        print("🔑 Saving to Keychain - Key: \(key)")
+        guard let data = value.data(using: .utf8) else {
+            print("❌ Failed to convert value to data")
+            return false
+        }
+
+        let query: [String: Any] = [
+            kSecClass as String: kSecClassGenericPassword,
+            kSecAttrAccount as String: key,
+            kSecValueData as String: data
+        ]
```

Comment:
It's better to avoid using force unwrapping because it can cause the app to crash:
```suggestion
        var query: [CFString: Any] = [
            kSecClass: kSecClassGenericPassword,
            kSecAttrAccount: key
        ]
        if let valueData = value.data(using: .utf8) {
            query[kSecValueData] = valueData
        }
```